scripts/startLocalCluster.sh

```diff
@@ -133,6 +137,34 @@ for aid in `seq 0 $(( $NRAGENTS - 1 ))`; do
     [ "$INTERACTIVE_MODE" == "R" ] && sleep 1
     PORT=$(( $AG_BASE + $aid ))
     AGENCY_ENDPOINTS+="--cluster.agency-endpoint $TRANSPORT://$ADDRESS:$PORT "
+    if [ "$AUTOUPGRADE" == "1" ];then
+      $ARANGOD \
+          -c none \
+          --agency.activate true \
+          --agency.compaction-step-size $COMP \
+          --agency.compaction-keep-size $KEEP \
+          --agency.endpoint $TRANSPORT://$ENDPOINT:$AG_BASE \
+          --agency.my-address $TRANSPORT://$ADDRESS:$PORT \
+          --agency.pool-size $NRAGENTS \
+          --agency.size $NRAGENTS \
+          --agency.supervision true \
+          --agency.supervision-frequency $SFRE \
+          --agency.supervision-grace-period 5.0 \
+          --agency.wait-for-sync false \
+          --database.directory cluster/data$PORT \
+          --javascript.enabled false \
+          --server.endpoint $TRANSPORT://$ENDPOINT:$PORT \
+          --server.statistics false \
+          --log.file cluster/$PORT.log \
+          --log.force-direct true \
+          --log.level $LOG_LEVEL_AGENCY \
+          --javascript.allow-admin-execute true \
+          $STORAGE_ENGINE \
+          $AUTHENTICATION \
+          $SSLKEYFILE \
+          --database.auto-upgrade true
```

Comment:
is this missing the backslash at the end of the line?

Reply:
Yes, will fix.